src/main/java/org/apache/sling/graphql/core/schema/RankedSchemaProviders.java

```diff
@@ -43,7 +43,7 @@
 @Component(service = RankedSchemaProviders.class)
 public class RankedSchemaProviders implements SchemaProvider {
 
-    final RankedServices<SchemaProvider> providers = new RankedServices<>(Order.ASCENDING);
+    final RankedServices<SchemaProvider> providers = new RankedServices<>(Order.DESCENDING);
```

Comment:
Could you just add a comment here that before this commit the order was the opposite?
I don't think it will break anything, but just in case.